README.md

```diff
@@ -35,9 +26,20 @@ If you did not use the sample database, you must run migrations and then downloa
 `deactivate`
 
 ## Environment variables
-There's an .env file in the same path as settings.py
-where all environment variables are and should be placed.
-You should *call* them in settings.py and then import them like settings.KEYWORD
+There's an .env file in the same path as settings.py where all environment variables are and should be placed. You should *call* them in settings.py and then import them like settings.KEYWORD
+
+`cp bankgreen/.env.template bankgreen/.env`
+
+## Database
+To setup the database, you must run migrations, add sample data by installing the initial fixture and download a list of countries and regions:   
+`python manage.py migrate`   
+`python manage.py loaddata fixtures/initial/initial.json`   
+`python manage.py cities_light`   
+
+Then create a superuser:   
+`python manage.py createsuperuser`
+
+To update initial fixture, run `python3 manage.py dumpdata <database>` and remove entries added to the database by the migrate command. Specifically, this means any entries for the 'django_content_type' table, which has a UNIQUE constraint on it's fields, but more generally, refers to any internal Django tables not explicitly defined in the various models.
```

Comment:
Could you please elaborate on this a little bit? What apps should actually be dumped and what explicit commands should be run?

`python manage.py dumpdata > fixtures/all.json` seems to not get enough. `python manage.py dumpdata brand > fixtures/brand/brand.json` seems to too much.